src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+import { msg } from './intl'
+
+class GlobalErrorBoundary extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = { hasError: false }
+  }
+
+  componentDidCatch (error, info) {
+    this.setState({ hasError: true })
+    console.error(error)
+  }
+
+  render () {
+    const title = msg.globalErrorBoundaryTitle()
+    const descr = msg.globalErrorBoundaryDescription()
+    if (this.state.hasError) {
+      // You can render any custom fallback UI
+      return (
+        <div>
+          <nav className='navbar obrand_mastheadBackground obrand_topBorder navbar-pf-vertical'>
+            <div className='navbar-header'>
+              <a href='/' className='navbar-brand obrand_headerLogoLink' id={`pageheader-logo`}>
+                <img className='obrand_mastheadLogo' src={branding.resourcesUrls.clearGif} />
+              </a>
+            </div>
+          </nav>
+          <div className='container text-center obrand_errorContainer'>
```

Comment:
once you have
`import style from './style.css'`
you can then do
```className={`${style['errorContainer']}```

(don't use the `obrand` prefix, since those are reserved for things that change between brands)